src/components/Modals/SeeInWorldModal/SeeInWorld.module.css

```diff
@@ -0,0 +1,52 @@
+.content {
```

Comment:
Should this file be renamed to `SeeInWorldModal.module.css`?

Reply:
I completely forgot to rename the files 🤦 , good catch.